retrievalmarket/types.go

```diff
@@ -397,8 +397,8 @@ func NewQueryV0(payloadCID cid.Cid) Query {
 
 // QueryResponse is a miners response to a given retrieval query
 type QueryResponse struct {
-	Status QueryResponseStatus
-	//PayloadCIDFound QueryItemStatus // V1 - if a PayloadCid was requested, the result
+	Status        QueryResponseStatus
+	PieceCIDFound QueryItemStatus // V1 - if a PieceCID was requested, the result
```

Comment:
this seemed to be a typo/leftover from the old version when Piece/Payload CID weren't distinct

Reply:
ah, right that was it.